modules/platform/nuxeo-drive-server/nuxeo-drive-core/src/test/java/org/nuxeo/drive/fixtures/AuditChangeFinderTestSuite.java

```diff
@@ -79,6 +79,25 @@
     @Inject
     protected TrashService trashService;
 
+    /** @since 11.3 */
+    @Test
+    public void testImpactedUser() {
+        log.trace("Register a sync root for Administrator");
+        nuxeoDriveManager.registerSynchronizationRoot(session.getPrincipal(), folder1, session);
+        txFeature.nextTransaction();
```

Comment:
You could have the newline here, as suggested initially, to separate the actions from the expected check, as in the other tests.